### PR TITLE
UI cleanup for campaigns and properties

### PR DESCRIPTION
**Pull request by coderberry**

### CODE REVIEW

File: app/models/concerns/impressionable.rb [hunk 1]
```diff
@@ -86,7 +86,7 @@ def daily_impressions_count(date = nil)
   end
 
   def daily_impressions_per_mille
-    daily_impressions_count / 1_000.to_f
+    daily_impressions_count.to_i / 1_000.to_f
```
Comment (hopsoft) [inline app/models/concerns/impressionable.rb]: Were these counts returning nil?

File: app/views/@shared/page_heading_navs/_advertisers.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+                      wrap_tag: :li,
+                      wrap_class: "nav-item hs-has-sub-menu u-header__nav-item" %>
+<%= active_link_to "Creatives", creatives_path,
+                      class: "nav-link u-header__nav-link",
+                      wrap_tag: :li,
+                      wrap_class: "nav-item hs-has-sub-menu u-header__nav-item" %>
+<%= active_link_to "Images", images_path(current_user.to_gid_param),
+                      class: "nav-link u-header__nav-link",
+                      wrap_tag: :li,
+                      wrap_class: "nav-item hs-has-sub-menu u-header__nav-item" %>
```
Comment (hopsoft) [inline app/views/@shared/page_heading_navs/_advertisers.html.erb]: Indentation is a little odd.

File: app/views/@shared/page_heading_navs/_publishers.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+                          wrap_tag: :li,
+                          wrap_class: "nav-item hs-has-sub-menu u-header__nav-item" %>
+<%= active_link_to "Properties", properties_path,
+                          class: "nav-link u-header__nav-link",
+                          wrap_tag: :li,
+                          wrap_class: "nav-item hs-has-sub-menu u-header__nav-item" %>
+<%= active_link_to "Distributions", "#",
+                          class: "nav-link u-header__nav-link",
+                          wrap_tag: :li,
+                          wrap_class: "nav-item hs-has-sub-menu u-header__nav-item" %>
```
Comment (hopsoft) [inline app/views/@shared/page_heading_navs/_publishers.html.erb]: Indentation is a little odd.

File: app/models/concerns/impressionable.rb (hunk 1 above)
Comment (coderberry) [reply app/models/concerns/impressionable.rb]: yes

**Pull request merged**